stories/Components/Snackbar/snackbar.scss

```diff
@@ -0,0 +1,80 @@
+/* src/components/Snackbar.css */
+.mg-snackbar-wrapper {
+  position: fixed;
+  top: 20px;
+  left: 50%;
+  display: flex;
+  justify-content: center;
+  transform: translate(-50%, -300%);
+  transition: transform 0.3s ease-in-out;
+
+  &__open {
+    transform: translate(-50%, 0%);
+  }
+}
+
+.mg-snackbar {
+  display: flex;
+  align-items: center;
+  background-color: white;
+  color: #333;
+  border-radius: 4px;
+  padding: 10px 20px;
```

Comment:
Not a blocker, but ideally these use our theme sass variables/tokens so that UNDRR has a more square look and PW is more rounded. 

The shortcut way would be to make use of 

- $mg-radius-tab
- $mg-padding-tab